Python-packages/covidcast-py/setup.py

```diff
@@ -22,7 +22,7 @@
     install_requires=[
         "pandas<2",
         "requests",
-        "delphi-epidata>=4.1.1",
+        "delphi-epidata>=4.1.2",
```

Comment:
```suggestion
        "delphi-epidata>=4.1.10",
```
`4.1.10` should be the next version of `delphi-epidata`

Reply:
since the next version has a breaking API change, shouldn't we be incrementing more than the minor version number?

Reply:
agree that we should bump a bigger number

even more, that's the [patch number being bumped](https://semver.org/)

Reply:
In even a fairly loose interpretation of the semantic versioning rules: yeah, we probably should be.  

At the same time, bumping the major number would belie the fact that this is a relatively tiny change in the scope of the rest of the repository...  Theres no new functionality and the only real effective change is to the signature of one public method, `Epidata.async_epidata()`.  I would also argue that method is probably ***very*** rarely used, though thats just based on a hunch -- i find it hard to believe that many (if any?) people are actually using it because it is poorly documented and it doesnt appear in established call chains, but i have no demonstrable evidence for this.  [It ***IS*** used [once in `covidcast-indicators`](https://github.com/cmu-delphi/covidcast-indicators/blob/39c02fa228fcb4de2b3f3d19469e862861351a18/nowcast/delphi_nowcast/epidata.py#L51), but we have full control over that code & execution.]

Still, your point stands and this change is worth more than a single increment of the patch-level version.

This would be a lot easier if the client and its versioning were decoupled from the rest of the API code, but we wont truly get that until we release and get adoption of `epidatpy`, which lives in its own repo.

This discussion properly belongs over at https://github.com/cmu-delphi/delphi-epidata/pull/1288 so i will copypasta much of this text there, and we can continue in the appropriate place.